### ISSUE TITLE
Title & Description attributes are not working while sharing to facebook & google plus

### ISSUE THREAD

**Issue by sreeharikmarar** [huacnlee/social-share-button#61]
data-title & data-desc attributes are not taking while sharing on facebook and google plus

when "facebook"
        SocialShareButton.openUrl("http://www.facebook.com/sharer.php?u=#{url}",popup)
when "google_plus"
        SocialShareButton.openUrl("https://plus.google.com/share?url=#{url}", popup)

**Comment by jkubacki** [comment]
:+1:

**Comment by Ammet** [comment]
I have same issue

**Comment by watahell** [comment]
It looks like for Facebook, you have to use Open Graph Markup now:
https://developers.facebook.com/docs/sharing/webmasters

**Comment by vrouth** [comment]
Any updates on this issue. I'm encountering the same problem for facebook?

**Comment by jtruzzi** [comment]
@vrouth i used this fb debug tool https://developers.facebook.com/tools/debug/sharing/ and i got it working. 
I had to add all these og attributes to my head tag
```html
<meta property="fb:app_id" content="<%= Rails.application.secrets.facebook_app_id %>" />
<meta property="og:url" content="<%= request.original_url %>" />
<meta property="og:type" content="article" />
<meta property="og:title" content="Title" />
<meta property="og:description" content="Description" />
<meta property="og:image" content="image full url" />
```
Also it seems to stop working if you have line breaks on your description.

**Comment by coin3d** [comment]
Strange thing is that appending the title and description attribute to the sharer.php URL still works - see this answer for example http://stackoverflow.com/a/40319441

**Comment by huacnlee** [comment]
https://developers.google.com/+/web/share/

In this page, I can't find any about custom option of Google+ share title.

Facebook just fixed by 0.8.7

**Comment by nik-007** [comment]
how to share title text on google plus there is no any such thing also why there is no whats app share icon here?
